security/cosign/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           golang 1.0
 
 go.setup            github.com/sigstore/cosign 2.5.0 v
-revision            0
+revision            1
```

Comment:
It sounds like this variant could never successfully build with the latest version of the package. So then there is no need to increase the "revision".